test/sidebar-additional-tabs.html

```diff
@@ -0,0 +1,120 @@
+    <html>
+        <head>
+            <meta charset="UTF-8" />
+            <meta name="viewport" content="width=device-width, initial-scale=1">
+            <title>Sidebar Test Page</title>
+            <style>
+                body,
+                html,
+                .sidebar {
+                    background: #ccc;
+                    border: 0 none;
+                    box-sizing: border-box !important;
+                    height: 100%;
+                    margin: 0;
+                    padding: 0;
+                    width: 100%;
+                }
+
+                .sidebar {
+                    display: flex;
+                }
+
+                .inputs {
+                    padding: 15px;
+                }
+
+                .inputs input {
+                    margin-bottom: 10px;
+                }
+            </style>
+        <link rel="stylesheet" type="text/css" href="../dev/en-US/sidebar.css" />
+        <script src='https://unpkg.com/react/umd/react.development.js'></script>
+        <script src="../dev/en-US/sidebar.js"></script>
+        </head>
+        <body>
+            <div class="sidebar">
+                <div class="inputs">
+                    <div>
+                        <label>
+                            <div>File Id</div>
+                            <input class="file" type="text" placeholder="Enter file id" />
+                        </label>
+                        <label>
+                            <div>Auth Token</div>
+                            <input class="token" type="text" placeholder="Enter auth token" />
+                        </label>
+                    </div>
+                    <button type="button" onclick="load(true)">Submit</button>
+                </div>
+                <div class="preview sidebar"></div>
+            </div>
+            <script>
+                const Tab = function(contents) {
+                    return React.createElement('div', null, contents);
+                }
+                function load(refreshOnComplete = false) {
+                    const { ContentSidebar } = Box;
+                    const token = document.querySelector('.token').value || localStorage.getItem('token');
+                    const fileId = document.querySelector('.file').value || localStorage.getItem('file');
+
+                    if (!token || !fileId) {
+                        return;
+                    }
+
+                    localStorage.setItem('token', token);
+                    document.querySelector('.token').value = token;
+
+                    localStorage.setItem('file', fileId);
+                    document.querySelector('.file').value = fileId;
+
+                    const placeholder = document.createElement('div');
+                    placeholder.textContent = 'test';
+
+                    const sidebar = new ContentSidebar();
+                    sidebar.show(fileId, token, {
+                        hasAdditionalTabs: true, 
```

Comment:
hasRecommendedApps and recommendedApps?

Reply:
Wanted to keep it generic in case a platform customer/user uses it for something that's not recommended apps. Could be useful to have some extra buttons in the sidebar nav that do custom stuff.